packages/platform-sdk-profiles/src/drivers/electron/server/handlers.ts

```diff
@@ -0,0 +1,66 @@
+import { Events } from "../events";
+import { Environment } from "../../../environment/env";
+import { IProfile } from "../../../contracts";
+
+const { ipcMain } = require("electron");
+
+export class Handlers {
+	readonly #env: Environment;
+
+	public constructor(env: Environment) {
+		this.#env = env;
+	}
+
+	public registerHandlers() {
```

Comment:
This is the handler registration that should take care of proxying all incoming calls on the main to the in memory driver.